### ISSUE TITLE
OpenToonz Terms of Use: little fix

### ISSUE THREAD

**Issue by kompowiec** [opentoonz/opentoonz#3433]
I discovered a caveat:

> (3) unable to offer the service due to act of God

in English it doesn't sound right. The correct expression which you probably did you mean it force majeure:

https://en.wikipedia.org/wiki/Force_majeure

**Comment by RodneyBaker** [comment]
Hi @kompowiec 

It's an interesting suggestion but I'm not convinced it is a fix.

In English, 'act of God' reads very clearly whereas 'Force Majeure' (derived from the French) has to be looked up to make sure we fully understand its definition and implication.

Other terminology that could work:
- Extenuating circumstance
- Providentially hindered
- Matters outside of reasonable control

A problem with all of these terms is that someone has to make the call on whether the reason for failing to meet the obligation was reasonable due to extenuating circumstances or due to failure, omission or neglect. 

More humorously...  of course in a legal document the more obscure the terminology apparently the better as that then requires legal minds to interpret.  That's rather the opposite of making things understandable in plain English.  ;)

**Comment by kompowiec** [comment]
English uses a lot of loanwords from different languages, e.g. spanish libre as free instead and it doesn't seem like a mistake. By contrast, a native speaker claimed that it was unable to understand it and rather it resembles arguments about god and the CIA by Terry a Davis than clause of the law :)

**Comment by RodneyBaker** [comment]
Always consider the source.  :)

**Comment by jpturcotte** [comment]
"Act of God" is the correct legal term, as a cursory Google search would have shown you.

**Comment by artisteacher** [comment]
To improve the grammar, it should probably be "due to an act of God.

**Comment by kompowiec** [comment]
ok i check and there is such an expression but with a slightly different context:

https://en.wikipedia.org/wiki/Act_of_God

> By contrast, other extraordinary man-made or political events are deemed force majeure.[5]

the border seems almost interchangeable. "Act of god" mean literally something beyond human capacity. You can close.
